### ISSUE TITLE
gnome-boxes

### ISSUE THREAD

**Issue by PVermeer** [ublue-os/bazzite#816]
### Describe the package

I'd like to request the package `gnome-boxes` because the flatpak stopped working with usb redirection. Containers also have troubles with polkit.

### Image

Desktop

**Comment by nicknamenamenick** [comment]
We probably won't ship this by default, but you can layer it to the image with:

```
rpm-ostree install gnome-boxes
```
Wait for it to layer it and update your image, then reboot and it is now part of your Bazzite image.

**Comment by PVermeer** [comment]
Yes, I have it layered. It's just something that (partially) doesn't work on flatpak and containers so only option is to layer it.

I mainly use it to update firmware on my Dual Sense controller on Windows. You can close this if its not planned ofc.
